example-deployment/README.md

```diff
@@ -350,6 +350,14 @@ deploy it
 .bin/minikube kubectl -- apply -f k8s/reporting.yaml
 ```
 
+## CronJobs
+
+deploy cronjobs
```

Comment:
Should we have an example on how to configure the job schedule, maybe a link to relevant docs in kubernetes and/or cron format.

Reply:
Yeah! that's a good idea! Thanks!